File: 2000 98 & ME/QuickStart/aspplus/samples/webforms/ctrlauth/CustomParsing/VB/Item.vb
Imports System
Imports System.Web
Imports System.Web.UI

Namespace CustomParsingControlSamples 

    Public Class ItemVB : Inherits Control

       Private _message As String

       Public Property Message As String
          Get
             Return _message
          End Get
          Set 
             _message = Value
          End Set
       End Property

    End Class

End Namespace

File: 2000 98 & ME/QuickStart/aspplus/samples/webforms/ctrlauth/CustomParsing/Item.cs
using System;
using System.Web;
using System.Web.UI;

namespace CustomParsingControlSamples {

    public class Item : Control {

       private String _message;

       public String Message {
          get {
             return _message;
          }
          set {
             _message = value;
          }
       }
    }
}